pkg/controller/core/queuedworkload_controller.go

```diff
@@ -75,11 +75,18 @@ func (r *QueuedWorkloadReconciler) Create(e event.CreateEvent) bool {
 	if wl.Spec.Admission == nil {
 		if !r.queues.AddOrUpdateWorkload(wl.DeepCopy()) {
 			log.V(2).Info("Queue for workload didn't exist; ignored for now")
+			if err := r.cache.UpdateWorkloadStatus(context.Background(), wl, kueue.QueuedWorkloadPending, "Pending", "Queue for workload didn't exist"); err != nil {
+				log.V(2).Error(err, "Failed to update QueuedWorkload Status")
```

Comment:
```suggestion
				log.V(2).Error(err, "Updating QueuedWorkload Status")
```

Reply:
the error message already starts with something like Error: in the logs, so saying "failed to" seems redundant

Reply:
got it